src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -134,11 +135,21 @@ public void processEventsInBatches(
    * @param events the event JSON objects to save.
    */
   public List<Event> saveAll(Collection<Event> events) {
```

Comment:
This method can be removed since it's no longer used.

Reply:
DONE - next push.